lab-kyle/app/controller/game-controller.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+// npm modules
+const angular = require('angular');
+// app modules
+const Monster = require('../model/monster.js');
+
+// module logic
+angular.module('demoApp').controller('GameController', [GameController]);
+
+function GameController(){
+  this.history = 'WELCOME TO FINDING A JOB!';
```

Comment:
Close. You're defining this.history here and treating it as an array in your controller methods. See the problem?